### PR TITLE
Resume Dashboard and Manage Attendance

### PR DESCRIPTION
**Pull request by aastikg20**

### CODE REVIEW

**Review by ManasGupta1310 — CHANGES_REQUESTED** [review]
Look at these reviews and get it done by tomorrow positively!

**Comment by ManasGupta1310** [comment]
Also these pages are of the admin side. Why did you put them up in student side?

File: pages/student/rc/[rcId]/ManageAttendance.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  
+  return (
+    <div style={{ padding: "0 2rem" }}><Meta title="Student Dashboard - Events" />
+    <Box component={Grid} boxShadow={10} sx={{
+      backgroundColor:'white',
+    }}>
+      <div style={{display:'flex',marginTop:'5%',
+        padding:'2%',justifyContent:'space-between'}}><Typography variant='h5' fontWeight='bold'>Manage Attendance</Typography>
+      <MoreVertIcon sx={{cursor:'pointer'}}></MoreVertIcon></div>
+      <TableContainer component={Paper}>
```
[the remark this answers is not in the document]
Comment (aastikg20) [reply pages/student/rc/[rcId]/ManageAttendance.tsx:34]: what's the difference?

[the remark this answers is not in the document]
Comment (1-Harshit) [reply pages/student/rc/[rcId]/ManageAttendance.tsx:34]: Consistency, theming

[the remark this answers is not in the document]
Comment (aastikg20) [reply pages/student/rc/[rcId]/ManageAttendance.tsx:34]: from next i will do with datagrid,can this be table, i will need to change whole code :(

[the remark this answers is not in the document]
Comment (1-Harshit) [reply pages/student/rc/[rcId]/ManageAttendance.tsx:34]: I clearly remember manas telling you to use data grid.
I also remember that i told you to go thru the existing code base.
Had you payed attention to either, this wouldn't have happened.

And no, Table won't merge

**Pull request closed without merging**